src/module/display-home.js

```diff
@@ -0,0 +1,42 @@
+async function getData() {
+  const response = await fetch('https://www.themealdb.com/api/json/v1/1/search.php?s');
+  const data = await response.json();
+  const header = document.querySelector('.head');
+  const random = document.querySelector('.random');
+  const listSection = document.querySelector('.home');
+  listSection.before(header);
```

Comment:
Thank you for your commits! Code is working fine and looks great. But I don't understand why you use .before in the display-home.js? Why you don't want just to change the order of tags  in index.html ?

Reply:
Thank you for your comment👍 , I did that because when I changed the order of tags in the HTML page they didn't arrange it correctly.